alkira/helper.go

```diff
@@ -62,14 +61,14 @@ func convertTypeListToStringList(in []interface{}) []string {
 	return strList
 }
 
-type getSegmentById = func(id string) (alkira.Segment, error)
+func convertSegmentIdsToSegmentNames(ids []string, m interface{}) ([]string, error) {
```

Comment:
If you want to, you could still pass `alkira.AlkiraClient` here, the function is specific for Alkira client related manipulation, we won't need any other metadata here...